src/supertux/sector.cpp

```diff
@@ -287,8 +289,10 @@ Sector::activate(const Vector& player_pos)
   }
 
   // Run init script
-  if (!m_init_script.empty() && !Editor::is_active()) {
+  if (!m_init_script.empty() && !Editor::is_active() &&
+      ((m_init_script_run_once && !m_init_script_run) || !m_init_script_run_once)) {
```

Comment:
I suppose this could be replaced with `(!m_init_script_run_once || !m_init_run_script)`.